learning_loop_node/detector/detector_node.py

```diff
@@ -75,6 +85,7 @@ async def soft_reload(self) -> None:
             Context(organization=self.organization, project=self.project),
             self.loop_communicator)
         self.relevance_filter = RelevanceFilter(self.outbox)
+        self.version_control = rest_version_control.VersionMode.FollowLoop
```

Comment:
```suggestion
        self.version_control = rest_version_control.VersionMode.Pause if os.environ.get(
            'VERSION_CONTROL_DEFAULT', 'follow_loop').lower() == 'pause' else rest_version_control.VersionMode.FollowLoop
```
Should this not also be set according to the environment variable?
I am not sure what the soft reload is meant for